### PR TITLE
Add mutation for updating projects

### PR DESCRIPTION
**Pull request by mafesan**
This feature allows to update Bestiary projects.

There is one new mutation: `updateProject`, which allows to update the name and the title of an existing project based on its ID.

Note: This PR is part of a set of changes coming from #43 . It depends on the changes from PR #44, where the Project model and the method for finding projects are defined.

### CODE REVIEW

File: releases/unreleased/update-projects.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+title: Update Projects
+category: added
+author: Miguel Ángel Fernández <mafesan@bitergia.com>
+issue: 42
+notes: >
+    This feature allows update existing Bestiary projects.
+
+    There is a new mutation: `updateProject`, which allows
+    to update the name and the title of Projects based on the
+    Project ID.
```
Comment (sduenas) [inline releases/unreleased/update-projects.yml:11]: Add an end line here.

**Comment by mafesan** [comment]
@sduenas I have addressed your comments and rebased this branch with `unicorn`, thank you!

**Comment by sduenas** [comment]
@mafesan can you rebase this branch to the latest changes?

**Comment by mafesan** [comment]
> @mafesan can you rebase this PR to the latest changes?

Done @sduenas !

**Comment by mafesan** [comment]
Done @sduenas, please check the changes I made in the last commit. Thank you!

**Pull request merged**